### PR TITLE
chapter 2 readme

### PR DESCRIPTION
**Pull request by ljj7975**

### CODE REVIEW

File: Chapter_2/README.md [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
-[get_rest_api_data.py](./rest/get_rest_api_data.py). 
+* [anaconda](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/anaconda) - <to be filled>
+* [csv_data](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/csv_data) - <to be filled>
+* [data_preprocessing](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/data_preprocessing) - <to be filled>
+* [data_set](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/data_set) - <to be filled>
+* [dockerfiles](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles) - <to be filled>
+* [google_scholar](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/google_scholar) - <to be filled>
+* [rest](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/rest) - <to be filled>
+* [visualization](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/visualization) - <to be filled>
```
Comment (ljj7975) [inline Chapter_2/README.md]: @leninworld please fill in each of them

File: Chapter_2/dockerfiles/README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ### Docker based environment for Data Science
 
-- `Dockerfile` along with readme is located [here](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles/jupyter-notebook)
+* [jupyter-notebook](https:/`/github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles/jupyter-notebook) - <to be filled>
+
+- [cpu-jupyter-notebook](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles/cpu-jupyter-notebook) - <to be filled>
```
Comment (ljj7975) [inline Chapter_2/dockerfiles/README.md]: @leninworld please fill in each of them, also how are they different?

File: Chapter_2/dockerfiles/cpu-jupyter-notebook/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+@Lenin please update this read me
```
Comment (ljj7975) [inline Chapter_2/dockerfiles/cpu-jupyter-notebook/README.md]: Please update this read me.

Also, can you explain each docker file on each read me? (what are the differences?)

**Comment by ljj7975** [comment]
@leninworld what are bashrc files for docker files? do we need them?

File: Chapter_2/dockerfiles/README.md (hunk 2 above)
Comment (leninworld) [reply Chapter_2/dockerfiles/README.md]: cleaned up. CPU should not be there (it was something left out while I was working on)

File: Chapter_2/dockerfiles/cpu-jupyter-notebook/README.md (hunk 3 above)
Comment (leninworld) [reply Chapter_2/dockerfiles/cpu-jupyter-notebook/README.md]: same as the comment in the above one.

File: Chapter_2/README.md (hunk 1 above)
Comment (leninworld) [reply Chapter_2/README.md]: done

**Pull request merged**